addOns/alertFilters/src/main/java/org/zaproxy/zap/extension/alertFilters/OptionsGlobalAlertFilterPanel.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.alertFilters;
+
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import javax.swing.JCheckBox;
+import javax.swing.JOptionPane;
+import javax.swing.SortOrder;
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.model.OptionsParam;
+import org.parosproxy.paros.view.AbstractParamPanel;
+import org.parosproxy.paros.view.View;
+import org.zaproxy.zap.view.AbstractMultipleOptionsTablePanel;
+
+public class OptionsGlobalAlertFilterPanel extends AbstractParamPanel {
+
+    private static final long serialVersionUID = 1L;
+
+    private GlobalAlertFilterMultipleOptionsPanel alertFilterOptionsPanel;
+
+    private AlertFilterTableModel alertFilterModel = null;
+
+    public OptionsGlobalAlertFilterPanel() {
+        super();
+        initialize();
+    }
+
+    private void initialize() {
+        this.setName(Constant.messages.getString("alertFilters.global.options.title"));
+        this.setLayout(new GridBagLayout());
```

Comment:
Could use `BorderLayout`, then it just needs to add the `alertFilterOptionsPanel`. The `initialize` method could be removed.